modules/module-postgres-storage/src/migrations/scripts/1684951997326-init.ts

```diff
@@ -0,0 +1,139 @@
+import { migrations } from '@powersync/service-core';
+
+import { dropTables } from '../../utils/db.js';
+import { openMigrationDB } from '../migration-utils.js';
+
+export const up: migrations.PowerSyncMigrationFunction = async (context) => {
+  const {
+    service_context: { configuration }
+  } = context;
+  await using client = openMigrationDB(configuration.storage);
+
+  /**
+   * Request an explicit connection which will automatically set the search
+   * path to the powersync schema
+   */
+  await client.transaction(async (db) => {
+    await db.sql`
+      CREATE SEQUENCE op_id_sequence AS int8 START
+      WITH
+        1
+    `.execute();
+
+    await db.sql`
+      CREATE SEQUENCE sync_rules_id_sequence AS int START
+      WITH
+        1
+    `.execute();
+
+    await db.sql`
+      CREATE TABLE bucket_data (
+        group_id integer NOT NULL,
+        bucket_name TEXT NOT NULL,
+        op_id bigint NOT NULL,
+        CONSTRAINT unique_id PRIMARY KEY (group_id, bucket_name, op_id),
+        op text NOT NULL,
+        source_table TEXT,
+        source_key bytea,
+        table_name TEXT,
+        row_id TEXT,
+        checksum bigint NOT NULL,
+        data TEXT,
+        target_op bigint
+      )
+    `.execute();
+
+    await db.sql`CREATE TABLE instance (id TEXT PRIMARY KEY) `.execute();
+
+    await db.sql`
+      CREATE TABLE sync_rules (
+        id BIGSERIAL PRIMARY KEY,
```

Comment:
The size of this should match group_id used in other places (plain `SERIAL` which is equivalent to `INTEGER` instead of `BIGSERIAL`).

This also creates a second sequence, `sync_rules_id_seq`, in addition to the `sync_rules_id_sequence` explicitly defined. It would be better to either:
1. Change this to a plain INTEGER field, with either no default, or a default of `nextval('sync_rules_id_sequence')`, OR:
2. Remove the explit `sync_rules_id_sequence`, and just rely on the auto-incrementing id behavior of `SERIAL`.

Reply:
I've implemented `1` with no default.